public/locales/fr/translation.json

```diff
@@ -153,7 +153,7 @@
     "postalCode": "Code Postal",
     "sortingCode": "Code de tri",
     "state": "État/Province",
-    "userRegistrationRequired": "Inscription de l'utilisateur requise",
+    "Is Public": "est public",
```

Comment:
This is not in keeping with the camel case standard. Remove the space, use camel case and update the source code.